### PR TITLE
Clone apple website

### PR DESCRIPTION
**Pull request by raheebwa**
#### Features 
* CSS gradients
* CSS Flexbox
* CSS Grid

#### Others
* Linted using stylelint specification
* Validated HTML

### CODE REVIEW

File: css/application.css [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+.jumbo {
+  font-family:
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Helvetica,
+    Arial,
+    sans-serif;
+  background-image: url(/images/your_verse_hero.jpg);
+  background-position: bottom;
```
Comment (stratospherique) [inline css/application.css]: According to your repository, I see that the path that you are referencing for the background image is wrong. :pray: check it

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work :clap: . However, some changes needs to be implemented in order for us to approve your project. check below :arrow_down: 

- **[REQUIRED]** you linter configuration is not set properly, I  can't see any checks. :pray: fix it

- **[REQUIRED]** Run your html through the [validator](validator.w3.org) and fix all the errors.

- **[REQUIRED]** the background image is not showing up. :no_entry_sign: 

### That's it :relieved: . When finished, kindly resubmit another code review request :pray:

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work @raheebwa :clap: . You made all the necessary changes :100: except one. I still can't see the background image. :pray: fix it and submit another code review request.

File: css/application.css (hunk 1 above)
Comment (raheebwa) [reply css/application.css]: It has been fixed..

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap:  The project is approved :+1: 

You can merge it.
Good luck with the next project.

**Pull request merged**